dev/tools/gen_keycodes/data/windows_logical_to_window_vk.json

```diff
@@ -176,41 +176,5 @@
   "Zoom": ["ZOOM"],
   "Noname": ["NONAME"],
   "Pa1": ["PA1"],
-  "OemClear": ["OEM_CLEAR"],
```

Comment:
These values are not used because these keys are manually added in `keyboard_maps_code_gen`.